superset/models/core.py

```diff
@@ -36,6 +36,7 @@
     Column,
     create_engine,
     DateTime,
+    Enum,
```

Comment:
I think I forgot to remove this when I converted to a string.. @AAfghahi

Reply:
Yeah, it was failing a lint test, so I removed this.